contracts/src/StabilityPool.sol

```diff
@@ -524,8 +528,14 @@ contract StabilityPool is LiquityBase, IStabilityPool, IStabilityPoolEvents {
             emit ScaleUpdated(currentScale);
             // If there's no scale change and no pool-emptying, just do a standard multiplication
         } else {
-            newP = currentP * newProductFactor / DECIMAL_PRECISION;
+            uint256 errorFactor;
+            if (lastBoldLossError_Offset > 0) {
+                errorFactor = lastBoldLossError_Offset * newProductFactor / lastBoldLossError_TotalDeposits;
+            }
+            newP = (currentP * newProductFactor + errorFactor) / DECIMAL_PRECISION;
```

Comment:
This seems consistent with the previous implementation, i.e. making `P` larger (than without error correction).  But see my comment on L461 questioning whether that was correct in the first place.

Reply:
Answered above.